pages/checkout/access.tsx

```diff
@@ -101,6 +102,13 @@ const CheckoutAccessView: NextPage = () => {
 
   return (
     <>
+      <Head>
+        <title>
+          {intl.formatMessage({ id: "checkout.prefix.title" })}
+          -
```

Comment:
prefer unicode. i also see this snippet is repeated. maybe a move this to a utility function?

Reply:
Sure 👍